GDSCommon-Demo/GDSCommon-Demo/Mocks/MockDatePickerScreenViewModel.swift

```diff
@@ -8,6 +8,7 @@ struct MockDatePickerScreenViewModel: DatePickerScreenViewModel {
     var datePickerFooter: String? = "Example date picker footer"
     var buttonViewModel: ButtonViewModel
     var rightBarButtonTitle: GDSLocalisedString? = "Right bar button"
+    var backButtonIsHidden: Bool = false
```

Comment:
`let`?

Reply:
I could change it but it feels like outside the scope here